src/pages/Dashboard/NoTeamRegistration.js

```diff
@@ -0,0 +1,347 @@
+import React, { useState } from 'react';
+import { useMutation, gql } from '@apollo/client';
+import { Box, Text, Button, Center, Input, FormControl } from '@chakra-ui/react';
+
+const NoTeamRegistration = props => {
+  const [currentPage, setPage] = useState(0);
+  const [teamName, setTeamName] = useState('');
+  const [teamAffiliation, setTeamAffiliation] = useState('');
+  const [inputList, setInputList] = useState([{ email: '' }]);
+
+  const incrementPage = async e => {
+    e.preventDefault();
+    if (currentPage < 2) {
+      setPage(currentPage + 1);
+    }
+  };
+
+  const decrementPage = async e => {
+    e.preventDefault();
+    if (currentPage > 0) {
+      setPage(currentPage - 1);
+    }
+  };
+
+  let pages = [
+    <InitialNoTeamPage incrementPage={incrementPage} />,
+    <CreateTeamForm
+      incrementPage={incrementPage}
+      decrementPage={decrementPage}
+      teamName={teamName}
+      teamAffiliation={teamAffiliation}
+      setTeamName={setTeamName}
+      setTeamAffiliation={setTeamAffiliation}
+      inputList={inputList}
+      setInputList={setInputList}
+    />,
+    <ConfirmTeamCreation
+      decrementPage={decrementPage}
+      teamName={teamName}
+      teamAffiliation={teamAffiliation}
+      userId={props.userId}
+      inputList={inputList}
+    />,
+  ];
+
+  return pages[currentPage];
+};
+
+const InitialNoTeamPage = props => {
```

Comment:
great job abstracting each component! I'm thinking we can put each one in a separate file - it'll just be a bit more readable and easier to review, even though our folder hierarchy is already super messed.

maybe something like this under `src/components`

DashboardNoTeam
-- InitialPage
---- InitalPage.js
-- CreateTeamForm
---- CreateTeamForm.js
-- ConfirmTeamCreation
---- ConfirmTeamCreation.js